content/en/docs/reference/access-authn-authz/abac.md

```diff
@@ -9,6 +9,11 @@ content_template: templates/concept
 weight: 80
 ---
 
+{{< note >}}
+{{< feature-state state="deprecated" for_k8s_version="1.17" >}}
```

Comment:
I recommend putting the feature-state shortcode _before_ the ` {{< note >}}` opening shortcode.

Reply:
Is it possible to either use the `feature-state` shortcode or add the deprecation comment in the note?

Reply:
Not sure what you're asking there @kbhawkey

Reply:
I am questioning whether this page needs both the feature-state label _in addition to_ the note. The note provides more information.

Reply:
Page preview:
https://deploy-preview-16318--kubernetes-io-master-staging.netlify.com/docs/reference/access-authn-authz/abac/

Right, I _think_ the note is clear.